Source/Embeddings.Processing/FailedCompareEmbedding.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Dolittle.Runtime.Embeddings.Store;
+
+namespace Dolittle.Runtime.Embeddings.Processing
+{
+    /// <summary>
+    /// Exception that gets thrown when a failure occurs while comparing embedding.
+    /// </summary>
+    public class FailedCompareEmbedding : Exception
```

Comment:
It feels like we should be able to come up with a more descriptive name here.

Reply:
Yes I know :) Any suggestions?